hudi-tests-common/pom.xml

```diff
@@ -228,5 +228,15 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Could you move this up to the same group as `org.mockito`?  Also, it looks like `mockito-junit-jupiter` is already in the list.